ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vlan-interfaces.j2

```diff
@@ -68,7 +68,20 @@
 | Interface | VRF | IPv6 Address | IPv6 Virtual Address | Virtual Router Address | VRRP | ND RA Disabled | Managed Config Flag | IPv6 ACL In | IPv6 ACL Out |
 | --------- | --- | ------------ | -------------------- | ---------------------- | ---- | -------------- | ------------------- | ----------- | ------------ |
 {%         for vlan_interface in vlan_interfaces_ipv6 | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
-| {{ vlan_interface.name }} | {{ vlan_interface.vrf | arista.avd.default('default') }} | {{ vlan_interface.ipv6_address | arista.avd.default('-') }} | {{ vlan_interface.ipv6_address_virtual | arista.avd.default('-') }} | {{ vlan_interface.ipv6_virtual_router_address | arista.avd.default('-') }} | {{ vlan_interface.vrrp.ipv6 | arista.avd.default('-') }} | {{ vlan_interface.ipv6_nd_ra_disabled | arista.avd.default('-') | lower }} | {{ vlan_interface.ipv6_nd_managed_config_flag | arista.avd.default('-') | lower }} | {{ vlan_interface.ipv6_access_group_in | arista.avd.default('-') }} | {{ vlan_interface.ipv6_access_group_out | arista.avd.default('-') }} |
+{%             set row_vrf = vlan_interface.vrf | arista.avd.default('default') %}
+{%             set row_ip_addr = vlan_interface.ipv6_address | arista.avd.default('-') %}
+{%             set row_ip_vaddr = vlan_interface.ipv6_address_virtual | arista.avd.default('-') %}
+{%             if vlan_interface.ipv6_virtual_router_addresses is arista.avd.defined %}
+{%                 set row_varp = vlan_interface.ipv6_virtual_router_addresses | join(", ") %}
+{%             else %}
+{%                 set row_varp = vlan_interface.ipv6_virtual_router_address | arista.avd.default('-') %}
```

Comment:
```suggestion
{%                 set row_varp = '-' %}
```

Reply:
In the if condition we already did a defined test. So in else we can set its default value to '-' because it's not defined.